ant-cli/src/commands/file.rs

```diff
@@ -105,9 +134,20 @@ pub async fn upload(file: &str, public: bool, peers: NetworkPeers) -> Result<()>
     Ok(())
 }
 
-pub async fn download(addr: &str, dest_path: &str, peers: NetworkPeers) -> Result<()> {
-    let mut client = crate::actions::connect_to_network(peers).await?;
-    crate::actions::download(addr, dest_path, &mut client).await
+pub async fn download(
+    addr: &str,
+    dest_path: &str,
+    peers: NetworkPeers,
+    read_quorum: Option<ResponseQuorum>,
+    json: bool,
+) -> Result<()> {
+    let mut client_operation_config = ClientOperationConfig::default();
+    if let Some(read_quorum) = read_quorum {
+        client_operation_config.chunk_read_quorum(read_quorum);
```

Comment:
Not sure I understand why we would need a quorum for chunk? Isn't the default of 1 sufficient?

Reply:
Yes that is right, 1 is sufficient for general use case. But we would like to test out replication performance of the nodes and this would require us to tweak the read quorum.